src/linker/Linker.Steps/SweepStep.cs

```diff
@@ -390,37 +423,16 @@ static bool IsSecurityAttributeType (TypeDefinition definition)
 			if (definition == null)
 				return false;
 
-			if (definition.Namespace == "System.Security") {
-				return definition.FullName switch
-				{
-					// This seems to be one attribute in the System.Security namespace that doesn't count
-					// as an attribute that requires HasSecurity to be true
-					"System.Security.SecurityCriticalAttribute" => false,
```

Comment:
What this part removed intentionally?

Reply:
Yeah - this logic was requiring HasSecurity for all System.Security attributes except SecurityCriticalAttribute. Really, it is only required for SuppressUnmanagedCodeSecurityAttribute, or ones derived from System.SecurityAttribute - and the latter are represented as SecurityAttributes instead of CustomAttributes.